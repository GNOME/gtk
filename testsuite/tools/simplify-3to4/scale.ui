<interface>
  <object class="GtkScale">
  </object>
  <object class="GtkScale">
    <property name="draw-value">1</property>
  </object>
  <object class="GtkScale">
    <property name="draw_value">0</property>
  </object>
</interface>
