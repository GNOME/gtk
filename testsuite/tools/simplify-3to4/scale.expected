<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <requires lib="gtk" version="4.0"/>
  <object class="GtkScale">
    <property name="draw-value">1</property>
    <property name="visible">0</property>
  </object>
  <object class="GtkScale">
    <property name="visible">0</property>
    <property name="draw-value">1</property>
  </object>
  <object class="GtkScale">
    <property name="visible">0</property>
  </object>
</interface>
